docs/book/service-manager.md

```diff
@@ -110,6 +110,23 @@ Because the main filter is `Priority`, it can be set directly too:
 ];
 ```
 
+## PsrLoggerAbstractServiceFactory
+
+Same as above, you can use PsrLoggerAbstractServiceFactory to create a PSR-3 conform logger.
```

Comment:
* This is a different factory and we should add separate description. "Same as above" should be replaced.
* All class names must be set in backticks: `PsrLoggerAbstractServiceFactory`
* A link on "PSR-3 logger" with the target to "PSR-3 Support" page would be good here.